src/aboutDialog.ui

```diff
@@ -8,7 +8,7 @@
     <property name="type_hint">dialog</property>
     <property name="program_name">Flatseal</property>
     <property name="version">1.2.2</property>
-    <property name="copyright" translatable="yes">Copyright © 2020 - Martín Abente Lahaye</property>
+    <property name="copyright">Copyright © 2020 - Martín Abente Lahaye</property>
```

Comment:
Remove translatable (checked). But I also suggested to remove the accent on the `í` here (and leave appdata.xml text untouched).

Reply:
Done. I also removed the `í` accent on line 15.